app/controllers/registrations_controller.rb

```diff
@@ -22,7 +22,8 @@ def account_update_params
       :agency_id,
       :password,
       :password_confirmation,
-      :current_password
+      :current_password,
+      :approved
```

Comment:
I assume `approved` should only be a permitted attribute `if current_user.admin?`.  This looks insecure if I'm understanding this correctly.